README.md

```diff
@@ -42,6 +43,7 @@ positional arguments:
                         host server.This command will helps you to send all
```

Comment:
few lines above **inspect** is missing in the list ``{checks,deploy,migrate,move-masters-from}`` (this list appear twice)

Reply:
done !